Framework/PythonInterface/mantid/kernel/src/Exports/Unit.cpp

```diff
@@ -57,6 +58,15 @@ void export_Unit() {
 
   register_ptr_to_python<std::shared_ptr<Unit>>();
 
+  enum_<Mantid::Kernel::UnitParams>("UnitParams")
+      .value("l2", Mantid::Kernel::UnitParams::l2)
+      .value("twoTheta", Mantid::Kernel::UnitParams::twoTheta)
+      .value("delta", Mantid::Kernel::UnitParams::delta)
+      .value("efixed", Mantid::Kernel::UnitParams::efixed)
+      .value("difa", Mantid::Kernel::UnitParams::difa)
+      .value("difc", Mantid::Kernel::UnitParams::difc)
```

Comment:
Do we not want to give python access to uncalibrated difc? Obviously people can calculate the nominal difc from L1+L2 and tth but it might come in handy if people want a quick test of whether a workspace has been calibrated?